styles.css

```diff
@@ -0,0 +1,348 @@
+* {
+    margin: 0;
+    Padding: 0;
+    font-family: "Open Sans", helvetica, arial, sans-serif;
+    font-weight: lighter;
+}
+
+Img {
```

Comment:
You've got a typo here with the capitalization of `img` -- this style isn't being applied.